src/components/flat-table/flat-table-row/flat-table-row.style.ts

```diff
@@ -11,7 +11,6 @@ import { FlatTableProps } from "..";
 import { FlatTableRowProps } from "./flat-table-row.component";
 import addFocusStyling from "../../../style/utils/add-focus-styling";
 import { isSafari } from "../../../__internal__/utils/helpers/browser-type-check";
-import cellSizes from "../cell-sizes.style";
```

Comment:
My approach with this piece of work was to touch as little as possible as we've had situations with Flat Table where a simple change has been made and it's introduced more bugs than it fixed. I wanted to avoid that 😆  
Like you, I realised it was used somewhere else so it was left as is.